src/quintain-benchmark.c

```diff
@@ -207,22 +208,17 @@ int main(int argc, char** argv)
     if (json_object_get_boolean(
             json_object_object_get(json_cfg, "use_server_poolset")))
         work_flags |= QTN_WORK_USE_SERVER_POOLSET;
+
+    /* todo: make this an object, not a string, so we can buld up / time a
+     * sequence of operations */
+    op_str
+        = json_object_get_string(json_object_object_get(json_cfg, "operation"));
```

Comment:
I think some corresponding updates are needed in the block of `CONFIG_HAS_OR_CREATE` calls lower down in the file?  I tried running `tests/quintain-benchmark-example.json` in a local build and it filled in `bulk_direction` (which I think is eliminated by this PR) but did not populate `operation`.